posthog-core/src/index.ts

```diff
@@ -89,6 +90,7 @@ export abstract class PostHogCoreStateless {
     this.host = removeTrailingSlash(options?.host || 'https://app.posthog.com')
     this.flushAt = options?.flushAt ? Math.max(options?.flushAt, 1) : 20
     this.maxBatchSize = Math.max(this.flushAt, options?.maxBatchSize ?? 100)
+    this.maxQueueSize = options?.maxQueueSize ?? 1000
```

Comment:
should we log or throw here if maxQueueSize <= flushAt ??

i'd be tempted to have an even higher limit to reduce risk of dropping events but i guess we don't know what a reasonable number is 🤷

Reply:
All SDKs default to 1k right now, at least the ones I checked.
`maxBatchSize` also does not log out nor throw.
it's not that you `must` otherwise nothing works, it's just that does not make much sense.
when the timer that flushes periodically is triggered, it does not care if `flushAt` is reached yet and will flush all pending events anyway.

Reply:
I can do the `Math.max` though.